src/main.py

```diff
@@ -1,11 +1,16 @@
 import logging
 
-from fastapi import FastAPI
+from fastapi import FastAPI, Request, status
+from fastapi.encoders import jsonable_encoder
+from fastapi.exceptions import RequestValidationError
+from fastapi.responses import JSONResponse
 
 import auth.urls
 import database
+import elections.urls
 import officers.urls
 import permission.urls
+import tests.urls
```

Comment:
oops! This is from the merge of main into this branch; please remove this (was something weird I was doing, give it no mind)
```suggestion
```